### PR TITLE
WIP - Initial work for managing subscription suspensions

### PR DESCRIPTION
**Pull request by nlindblad**
- Checks whether the user is already signed in to an Identity account with a deliverable subscription

- Asks for subscription number, last name and post code if the user is not logged in or the Identity account does not have a valid deliverable subscription

- Presents the user with a date range picker that starts in two weeks time and have a max range of 12 weeks (configurable)

- Posts the final suspension request back to the app with the details needed to populate relevant third-party services

**Mobile**:
screen shot 2016-07-22 at 17 18 18

screen shot 2016-07-22 at 17 12 46

**Desktop**:
screen shot 2016-07-22 at 17 13 44

screen shot 2016-07-22 at 17 12 33

### CODE REVIEW

File: assets/javascripts/modules/suspend.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+define([
+    'modules/suspend/formElements'
+], function (
+    formElements
+) {
+    'use strict';
+    function init() {
+        if (formElements.$SUSPEND_FORM.length) {
+            require(['modules/suspend/suspendFields'], function(suspendFields) {
+                console.log('Loaded suspend JS stuff');
```
Comment (tomverran) [inline assets/javascripts/modules/suspend.js]: console.log

File: package.json [hunk 2]
```diff
@@ -39,6 +39,7 @@
     "lodash": "^4.13.1",
     "lodash-amd": "~3.10.0",
     "moment": "^2.13.0",
+    "moment-business-days":"0.0.10",
```
Comment (tomverran) [inline package.json]: There's a library for everything!

**Comment by tomverran** [comment]
👍

**Comment by paulbrown1982** [comment]
👍 too

**Pull request merged**